### PR TITLE
NC Kilderkin - bar/brewery ship

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
_"Spaceworthy bar/microbrewery with everything one needs to facilitate poor life choices: lots of booze, smokes, and lack of food."_, - Shipyard.
**Ship size:** Medium (264 tiles)
**Recommended crew:** 3
**Cost:** 42900
**Power Gen Type:** Plasma
**Expeditions:** No
**IFF Console:** No
**Available Roles:** Bartender, Botanist, Mercenary, Pilot

## Why / Balance
Dedicated bar ship. NSFD better start their printer for tickets for drunken driving.

## Technical details
.ymls

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame.

**Changelog**
:cl: erhardsteinhauer
- add: Added NC Kilderkin - bar/brewery ship.

### CODE REVIEW

**Review by TsjipTsjip — APPROVED** [review]
Facts:
- Ship price at shipyard: 42900
- Ship sale value at shipyard: 38754

Minor notes:
- Musician's corner is a bit dark. Moody tho, could argue keeping it like this.

This ship is a good example of a bar ship that is quite literally nothing else except for a little bit of botany, but it does not have a kitchen which keeps it true to its name. I see no issues with this ship as of now.

**Comment by ErhardSteinhauer** [comment]
> Facts:
> 
> * Ship price at shipyard: 42900
> * Ship sale value at shipyard: 38754

I think I went for 10% markup.

> Minor notes:
> 
> * Musician's corner is a bit dark. Moody tho, could argue keeping it like this.

There is a lamp in that corner for those who'd like more light.

>This ship is a good example of a bar ship that is quite literally nothing else except for a little bit of botany, but it does not have a kitchen which keeps it true to its name. I see no issues with this ship as of now.

Debated if I even should have botany there, but some cocktails require fruits or veggies.

**Comment by dvir001** [comment]
Ship mark up need to be 6% up to 15%
I think McCargo is 15%
Loader is 7%
And so on

**Comment by ErhardSteinhauer** [comment]
> Facts:
> 
> * Ship price at shipyard: 42900
> * Ship sale value at shipyard: 38754

Wait, do you mean despite me adding 10% markup it sells without one?

File: Resources/Prototypes/_NF/Shipyard/kilderkin.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  name: NC Kilderkin
+  description: "Spaceworthy bar/microbrewery with everything one needs to facilitate poor life choices: lots of booze, smokes, and lack of food."
+  price: 42900
+  category: Medium
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/kilderkin.yml
+
+- type: gameMap
+  id: kilderkin
+  mapName: 'NC Kilderkin'
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/_NF/Shipyard/kilderkin.yml:12]: Remove "NC" from the napName as NC should not appear in the IFF Tag.

File: Resources/Prototypes/_NF/Shipyard/kilderkin.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  mapPath: /Maps/Shuttles/kilderkin.yml
+  minPlayers: 0
+  stations:
+    kilderkin:
+      stationProto: StandardFrontierExpeditionVessel
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/_NF/Shipyard/kilderkin.yml:17]: StandardFrontierExpeditionVessel is for Expedition ships, use StandardFrontierVessel.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_NF/Shipyard/kilderkin.yml (hunk 1 above)
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Shipyard/kilderkin.yml]: Done

File: Resources/Prototypes/_NF/Shipyard/kilderkin.yml (hunk 2 above)
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Shipyard/kilderkin.yml]: And done

**Comment by ErhardSteinhauer** [comment]
Fixed shipyard file for the ship.

**Comment by Cheackraze** [comment]
lgtm. The engi locker is a little sus but as long as its not a hard suit locker, this looks just fine. I'm also ok with it having only a 10% markup since there isnt really anything on the ship to make it like, some kind of money printing ship.

**Comment by ErhardSteinhauer** [comment]
I think it's just engilocker without the hardsuit (hence the emergency wall locker in the same room). It's mostly there for looks. And for filled utility belt.

**Comment by ErhardSteinhauer** [comment]
Updated the ship to current standards and images in PR:
- Enabled Pilot Job.
- Replaced Engineer Locker with Utility Belt [Filled]
- Replaced Emergency EVA Wall Closet with Suit Wallstorage Unit (EVA).
- Swapped some consoles for tabletop versions.

**Comment by MagnusCrowe** [comment]
> Updated the ship to current standards and updated the images in PR:
> 
> * Enabled Pilot Job.
> * Replaced Engineer Locker with Utility Belt [Filled]
> * Replaced Emergency EVA Wall Closet with Suit Wallstorage Unit (EVA).
> * Swapped some consoles for tabletop versions.

Are you sure?  I just loaded it up and the pilot job isn't on the records computer.

Ship looks good.  I need to check with Checkraze about the camera server though, I'll get back to you.

**Comment by ErhardSteinhauer** [comment]
I'm fairly sure, yeah.

**Comment by MagnusCrowe** [comment]
> I'm fairly sure, yeah. 

Wild, I could have sworn I only saw 3 options last time I booted it up.  I see it now...I must be going insane.

No issues with the camera server.  Will probably be approving it today.

**Review by MagnusCrowe — APPROVED** [review]

**Pull request merged**